### PR TITLE
Fix text styles

### LINKED ISSUE #78: Contrast / Custom styles no-longer inherited correctly
Due to the _stronger_ styling rules on the text components, styles provided by either `contrast` or `styles` props are overwritten.

image

image

### PR DESCRIPTION
**Pull request by breadadams**
Close #78

### CODE REVIEW

**Comment by Kikobeats** [comment]
Please merge master to fix travis errors!

File: packages/react-microlink/src/components/Card/CardText.js [hunk 1]
```diff
@@ -3,13 +3,15 @@ import styled from 'styled-components'
...
 const Clamp = ({children, className, lines}) => (
-  <NanoClamp className={className} lines={lines} text={children} is='p' />
+  <NanoClamp className={className} lines={lines} text={children} is="p" />
 )
 
 const CardText = styled(Clamp)`
-  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
-  color: #181919;
-  margin: 0;
+  &&& {
```
Comment (Kikobeats) [inline packages/react-microlink/src/components/Card/CardText.js]: is this a valid syntax? what it means?

File: packages/react-microlink/src/components/Card/CardWrap.js [hunk 2]
```diff
@@ -48,6 +51,8 @@ const style = css`
   border-style: solid;
   border-color: #E1E8ED;
   overflow: hidden;
+  color: #181919;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (Kikobeats) [inline packages/react-microlink/src/components/Card/CardWrap.js]: maybe a good moment for extract `font-family` value and colors into a constants file 🤔

File: packages/react-microlink/src/components/Card/CardText.js (hunk 1 above)
Comment (breadadams) [reply packages/react-microlink/src/components/Card/CardText.js]: Yeah, it's _a thing_ haha.

In scss `&` references "itself", so chaining `&`, chains the selector.

```
.title {
  &&& {
    ...
  }
}

===

.title.title.title { ... }
```

Basically a non-invasive way of doing `!important`, cause you can overwrite these styles with a stronger selector (without needing `!important`).

Note, this doesn't work for selectors that can't be chained, ie. `h1` => `h1h1h1` wouldn't work

It's mentioned a bit in styled-components [docs](https://www.styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity)

File: packages/react-microlink/src/components/Card/CardWrap.js (hunk 2 above)
Comment (breadadams) [reply packages/react-microlink/src/components/Card/CardWrap.js]: Well, those values specifically are only used once I believe. But I'll open an issue to move colors to a file as you mention, cause the empty-state/loader could benefit from it 👌

File: packages/react-microlink/src/components/Card/CardText.js (hunk 1 above)
Comment (Kikobeats) [reply packages/react-microlink/src/components/Card/CardText.js]: nice!

**Pull request merged**